config/testgrids/kubernetes/sig-storage/config.yaml

```diff
@@ -71,6 +71,12 @@ dashboards:
       description: storage snapshot e2e tests for master branch
       alert_options:
         alert_mail_to_addresses: kubernetes-sig-storage-test-failures@googlegroups.com
+    - name: kind-master-alpha-beta
+      test_group_name: ci-kubernetes-e2e-kind-alpha-beta-features
+      base_options: include-filter-by-regex=Volume%7Cstorage
+      description: Runs storage e2e tests with no special requirements other than alpha or beta feature gates in a KinD cluster where alpha and beta feature gates and APIs are enabled.
```

Comment:
Nit: KinD to "Kind"?

Does this include Slow tests too?

Reply:
At the csi meeting we decided to add a alpha beta Slow job, after VAC GAed that job maybe empty but it should be okay.

Reply:
I follow the same style as [sig-testing/kubernetes-kind.yaml](https://github.com/kubernetes/test-infra/blob/1ecf273d004b0193c05599572a0cac896cd46e16/config/jobs/kubernetes/sig-testing/kubernetes-kind.yaml)

https://cs.k8s.io/?q=KinD&i=nope&literal=nope&files=&excludeFiles=&repos=kubernetes/test-infra

If we need to change them, I can do it in a follow-up PR. cc @BenTheElder @sunnylovestiramisu